### ISSUE TITLE
uninstall

### ISSUE THREAD

**Issue by Tamshi27** [UnamSanctam/SilentCryptoMiner#1830]
Hey, just wanted to ask how i can manually deinstall the miner from a pc without the uninstall.exe

**Comment by UnamSanctam** [comment]
Yes it's possible, though can be a bit difficult depending on the settings: https://github.com/UnamSanctam/SilentCryptoMiner/wiki#how-do-i-remove-the-miner.
